src/main/resources/static/js/tutorial/tutorial-header.js

```diff
@@ -0,0 +1,207 @@
+$(document).ready(function (){
+// tutorial code 삽입
+
+// 하이라이트할 요소들 생성
+let $button1 = $(document.getElementById("proj_list"));
+
+// 하이라이트할 요소들의 배열 (button 1, 2, 3, 4)
+let array_highlights = [$button1];
+// 툴팁이 보여줄 제목 및 내용의 배열
+let array_tooltip_data = [
+  { title: "Project list 이동", content: "Project list 로 이동합니다." }
+];
+
+// vail 생성
+let $vailUp = $('<div id="vail_up"></div>');
+let $vailDown = $('<div id="vail_down"></div>');
+let $vailLeft = $('<div id="vail_left"></div>');
+let $vailRight = $('<div id="vail_right"></div>');
+
+// tooltip 생성
+let $tooltip = $(
+  `<div id="tooltip">
+    <div id="tooltip_title_container">
+      <div id="tooltip_title"></div>
+      <button id="button_close_tooltip">X</button>
+    </div>
+    <p id="tooltip_content"></p>
+    <div>
+    </div>
+  </div>`
+);
+
+$("#proj_list").prepend($button1);
+
+// body 태그 내에 vail, tooltip들 넣기
+$("body").append($vailUp);
+$("body").append($vailDown);
+$("body").append($vailLeft);
+$("body").append($vailRight);
+$("body").append($tooltip);
+
+// vail, tooltip 기본 style 설정
```

Comment:
Please change the comments to English

Reply:
Do you mean all the code in js file!?

Reply:
f70b71cb72ebf79444cffac925457a37a3684084